action.yml

```diff
@@ -17,6 +17,10 @@ inputs:
     description: "Download op-reth"
     required: false
     default: ""
+  docker-compose:
```

Comment:
`docker compose` is not another tool to download but it gets downloaded if you are downloading the playground.